grafana/grafana
# Jaeger: Add support for variables

**What this PR does / why we need it**:

This PR adds support for variables in Jaeger queries.

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana/issues/44422

**Special notes for your reviewer**:

To test;
1. Create new dashboard with Jaeger as datasource,
2. Add variables for service, operation, tags, min duration, max duration,
3. Test that those variables are interpolated for dashboard queries,
4. Hover over the dashboard panel, select explore and ensure that those variables are also interpolated in explore
